internal/cmd/run/run.go

```diff
@@ -147,6 +147,7 @@ func Command() *cobra.Command {
 
 	// Deprecated flags
 	_ = sc.Fset.MarkDeprecated("uploadTimeout", "please use --upload-timeout instead")
+	_ = sc.Fset.MarkDeprecated("test-env-silent", "please remove it")
```

Comment:
Should include a better description! This flag no longer does anything, so simply say that. "please remove it" leaves the user rather clueless.

Reply:
😂 I feel nothing to say about this flag.